src/app/lib/utilities/getEmbedUrl/index.js

```diff
@@ -20,12 +22,19 @@ const shouldOverrideMorphEnv = (queryString, type) => {
   return isMediaType;
 };
 
+const getBaseUrl = isAmp => {
+  if (isLive() && isAmp) return LIVE_AMP_URL;
+  if (isAmp) return TEST_AMP_URL;
+  if (isLive()) return LIVE_CANONICAL_URL;
+  return TEST_CANONICAL_URL;
```

Comment:
How about:

```suggestion
 if (isAmp) {
   return isLive() ? LIVE_AMP_URL : TEST_AMP_URL;
 }
  return isLive() ? LIVE_CANONICAL_URL : TEST_CANONICAL_URL;
```

Or something along those lines. Or there could be another way, using a MAP / Array / Object to store the values.. Mmm 🤔

Reply:
Yea nice. I knew this conditional was naaarsty, was gonna revisit - I like this suggestion